lib/index.js

```diff
@@ -219,7 +218,11 @@ MapboxGeocoder.prototype = {
       // parse coordinates
       var coords = searchInput.split(/[\s(,)?]+/).map(function(c) {
         return parseFloat(c, 10);
-      });
+      }).reverse();
+
+      // client only accepts one type for reverseGeocode, so
+      // use first config type if one, if not default to poi.landmark
+      config.types ? [config.types[0]] : ["poi.landmark"];
```

Comment:
nit picking here but according to https://docs.mapbox.com/api/search/#reverse-geocoding `poi.landmark` is deprecated, returning the same as `poi`, so probably better to just change this to `poi`.